tests/integration-tests/tests/scheduler_plugin/test_scheduler_plugin.py

```diff
@@ -533,3 +535,54 @@ def _test_cluster_update(cluster, cluster_config):
     cluster.update(str(cluster_config), force_update="true")
     cluster.start()
     _check_fleet_status(cluster, "RUNNING")
+
+
+def _test_invoke_scheduler_plugin_event_handler_script(command_executor, compute_node, run_as_user):
+    """Test invoke-scheduler-plugin-event-handler.sh."""
+    # Create a dummy cluster config for test, echo "dummy config {Event} executed" during handler execution.
+    cluster_config = "/opt/parallelcluster/shared/cluster-config.yaml"
+    dummy_cluster_config = "/opt/parallelcluster/shared/dummy-cluster-config.yaml"
+    command_executor.run_remote_command(
+        f"sudo sed 's/echo \\\\\"/echo \\\\\"dummy config /g' {cluster_config} | sudo tee "
```

Comment:
are all of these escaping `\\\\\` needed?

Reply:
`\\\\\"` is to replace `echo \"` in the code to be `echo \"dummy`
We can not remove some of `\` to be for example `\\\"`, when run the remote command, `\\` is ignored, so we need extra `\\` before `\\`